.github/workflows/prod-deploys-create-issue.yml

```
@@ -0,0 +1,21 @@
+name: Create issue for push to prod
+on:
+  schedule:
+    # Runs on the 1st and 15th of each month at 9AM
+    - cron: "0 9 1,15 * *"
+
+jobs:
+  create_issue:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Create issue
+        env:
+          label: "push to prod"
+          title: ":rocket: Deploy to production: ${{ github.event.repository.updated_at }}"
+          body: "Deploy to production: ${{ github.event.repository.updated_at }}"
+        run: |
+          gh issue create --title "$title" --body "$body" --label "$label"
```

```suggestion
      - name: Set current date as env variable
        run: |
          echo "now=$(date +'%Y-%m-%d')" >> GITHUB_ENV

      - name: Create issue
        env:
          label: "push to prod"
        run: |
          title=":rocket: Deploy to production: ${now}"
          body="Deploy to production: ${now}"

          gh issue create --title "${title}" --body "${body}" --label "${label}"
```

implementation of above suggestion